frontends/web/src/components/view/view.tsx

```diff
@@ -85,9 +85,13 @@ export const View = ({
       )}
       {withBottomBar && (
         <div style={{ marginTop: 'auto' }}>
-          <Footer>
+          <footer className={style.footer}>
             <SwissMadeOpenSource />
-          </Footer>
+            <div className="m-right-half hide-on-small">
+              <Version />
+            </div>
+            <LanguageSwitch />
+          </footer>
```

Comment:
Hm, not rly a big deal but instead of sort of duplicating `Footer` here, would you think it makes sense to just re-style the `Footer` (to not use the inline `flex`, `flex-row`, `flex-end`,etc)? 

As it seems like this is the only difference between the `Footer` component and what we have above? 🤔